client/src/components/Navigation.js

```diff
@@ -0,0 +1,17 @@
+import "./Navigation.css";
+import { NavLink } from "react-router-dom";
+import { CgMenuGridR } from "react-icons/cg";
+import { BsSearch } from "react-icons/bs";
+
+export default function Navigation() {
+  return (
+    <div className="Navigation">
+      <NavLink to="/menu">
+        <CgMenuGridR color="#d84064" size="40px" />
+      </NavLink>
+      <NavLink to="/intro">
+        <BsSearch color="#d84064" size="35px" />
```

Comment:
is `/intro`your root/home page? if so you could name it better like this and at this stage of your app I don't really understand that it is a magnifier icon because this is normally used to indicate a search function or something like this.

Reply:
Yep, I am aware it is confusing. The feature should be removed as I did not have a clear idea how to setup the user experience and followed blindly the initial design.